easylist_adult/adult_adservers.txt

```diff
@@ -16,6 +16,7 @@
 ||adsturn.com^$third-party
 ||adult3dcomics.com^$third-party
 ||adultforce.com^$third-party
+||adultnetworksc.com^$third-party
```

Comment:
No need to make 3p

![image](https://user-images.githubusercontent.com/44526987/160457470-ba5c5ed0-a39f-4232-81c6-07109ea6f00b.png)